week-2/F-logical-operators/test.js

```diff
@@ -0,0 +1,23 @@
+const assert = require('assert')
+
+function Check (username, usertype){
+
+    if(usertype === "admin" || usertype === "manager"){
+        return "Username valid";
+    } else if (username[0] === username[0].toUpperCase() && username.length >= 5 && username.length <= 10)
+    {
+        return "Username valid";        
+    } else  {
+        return "Username invalid";        
+     
+    }
+
+}
+
+
+
+
+assert(Check("Al23", "managr")==="Username valid");
+assert(Check("Alexey23", "managr")==="Username valid");
+assert(Check("alexel23", "managr")==="Username invalid");
+
```

Comment:
Could you explain to me why you think these tests should pass?

Reply:
First test will be failed, there should be "manager"  usertype to pass.
Second  test got a proper username so it will pass all username is valid
Third got the wrong username with lowercase first letter and will be invalid  and test will be passed